### PR TITLE
Update Chinese translations to fix full/half-width issues

### PR DESCRIPTION
**Pull request by vadz**
Also make the terminology used more consistent.

@hjiawei Could you please check this one now? TIA!

cc @obmij

### CODE REVIEW

File: locale/zh_CN.po [hunk 1]
```diff
@@ -2094,94 +2095,94 @@ msgstr "无法从文件 '%s' 中读取图像 %%d。"
...
 #: ../src/common/imagbmp.cpp:487
 msgid "BMP: Couldn't write data."
-msgstr "BMP: 无法写数据。"
+msgstr "BMP：无法写数据。"
 
 #: ../src/common/imagbmp.cpp:533
 #, c-format
 msgid "BMP: header has biClrUsed=%d when biBitCount=%d."
-msgstr "BMP: 当 biBitCount=%d 时，文件头为 biClrUsed=%d。"
+msgstr "BMP：当 biBitCount=%d 时，表头則為 biClrUsed=%d。"
```
Comment (hjiawei) [inline locale/zh_CN.po:2132]:
```suggestion
msgstr "BMP：当 biBitCount=%d 时，表头则为 biClrUsed=%d。"
```

File: locale/zh_CN.po [hunk 2]
```diff
@@ -2416,18 +2417,18 @@ msgstr "前則消息重复一次。"
 #, c-format
 msgid "The previous message repeated %u time."
 msgid_plural "The previous message repeated %u times."
-msgstr[0] "The previous message repeated %u time."
+msgstr[0] "前則消息已重复 %u 次。"
```
Comment (hjiawei) [inline locale/zh_CN.po:2420]:
```suggestion
msgstr[0] "前则信息已重复 %u 次。"
```

File: locale/zh_CN.po [hunk 3]
```diff
@@ -5090,7 +5091,8 @@ msgstr "WebKit v1 不支持检索 JavaScript 脚本输出"
 msgid ""
 "GTK+ installed on this machine is too old to support screen compositing, "
 "please install GTK+ 2.12 or later."
-msgstr "本机安装的GTK+版本过低，不支持屏幕组合，请安装GTK+2.12或者更新版本。"
+msgstr ""
```
Comment (hjiawei) [inline locale/zh_CN.po:5094]: Will this empty string get an error or it will be ignored?

File: locale/zh_CN.po [hunk 4]
```diff
@@ -7329,7 +7331,7 @@ msgstr "角"
 
 #: ../src/richtext/richtextborderspage.cpp:575
 msgid "Corner &radius:"
-msgstr "角半径 (&r):"
+msgstr "角&半径："
```
Comment (hjiawei) [inline locale/zh_CN.po:7334]:
```suggestion
msgstr "角半径(&R)："
```

File: locale/zh_CN.po [hunk 5]
```diff
@@ -7849,7 +7851,7 @@ msgstr "勾选使用从右到左文字布局。"
 
 #: ../src/richtext/richtextfontpage.cpp:320
 msgid "Suppress hyphe&nation"
-msgstr "抑制断字 (&n)"
+msgstr "抑制断字"
```
Comment (hjiawei) [inline locale/zh_CN.po:7854]:
```suggestion
msgstr "抑制断字(&N)"
```

**Comment by hjiawei** [comment]
I got some small suggestions and one question for `zh-CN.po`. Not reviewed `zh-TW.po`. LGTM overall. Thanks all!

File: locale/zh_CN.po (hunk 1 above)
Comment (obmij) [reply locale/zh_CN.po:2132]: fixed in .po

File: locale/zh_CN.po (hunk 2 above)
Comment (obmij) [reply locale/zh_CN.po:2420]: fixed in .po

File: locale/zh_CN.po (hunk 3 above)
Comment (obmij) [reply locale/zh_CN.po:5094]: Screen Shot 2022-07-11 at 10 41 24

File: locale/zh_CN.po (hunk 4 above)
Comment (obmij) [reply locale/zh_CN.po:7334]: fixed in .po

File: locale/zh_CN.po (hunk 5 above)
Comment (obmij) [reply locale/zh_CN.po:7854]: fixed in .po

File: locale/zh_CN.po (hunk 2 above)
Comment (obmij) [reply locale/zh_CN.po:2420]: info --> 信息
message --> 消息
Screen Shot 2022-07-11 at 10 33 19
Screen Shot 2022-07-11 at 10 34 18
Screen Shot 2022-07-11 at 10 35 52

File: locale/zh_CN.po (hunk 3 above)
Comment (obmij) [reply locale/zh_CN.po:5094]: 5094 will be just an empty line. not even a quotation mark will be seen.

File: locale/zh_CN.po (hunk 1 above)
Comment (obmij) [reply locale/zh_CN.po:2132]: Found something weird... wondered why zh_CN.po leverage zh_TW.po after updating from .pot file.
Screen Shot 2022-07-11 at 10 55 29

File: locale/zh_CN.po (hunk 5 above)
Comment (obmij) [reply locale/zh_CN.po:7854]: Screen Shot 2022-07-11 at 11 00 58

File: locale/zh_CN.po (hunk 4 above)
Comment (obmij) [reply locale/zh_CN.po:7334]: Screen Shot 2022-07-11 at 11 02 19

File: locale/zh_CN.po (hunk 2 above)
Comment (obmij) [reply locale/zh_CN.po:2420]: Screen Shot 2022-07-11 at 11 05 00

File: locale/zh_CN.po (hunk 4 above)
Comment (obmij) [reply locale/zh_CN.po:7334]: comment revised to "角半径(&r)：" to be accurate.
Screen Shot 2022-07-11 at 11 18 14

File: locale/zh_CN.po (hunk 5 above)
Comment (obmij) [reply locale/zh_CN.po:7854]: comment revised.
Screen Shot 2022-07-11 at 11 21 44

File: locale/zh_CN.po (hunk 2 above)
Comment (hjiawei) [reply locale/zh_CN.po:2420]: Sorry, my bad. Updated suggestion. It is really `則` to `则` and nothing to do with `message`.

Comment (obmij) [reply locale/zh_CN.po:2420]: no worry had a global check and replaced accordingly.

**Review by hjiawei — APPROVED** [review]

**Comment by vadz** [comment]
Sorry, I'm confused because on one hand the version here was approved, but OTOH I received some more fixes via email -- which I've now pushed here as well. Just to be sure, the latest version, with the fixup commit, is the best one to merge, right?

**Review by hjiawei — APPROVED** [review]
The latest change LGTM. Re-approving based on the commit b80a57598feb1d194bf4e069d127e0cecacc03c7.

**Pull request closed without merging**